modules/rules_directory/0.0.0/MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+"Bazel dependencies"
+
+module(
+    name = "rules_directory",
+    version = "0.0.0",
```

Comment:
Is this intentional?

Reply:
I think the whole release is probably accidental, there's no code in the repo yet either.

Reply:
No, this was intentional, I was trying to send it to the BCR in parallel with a PR with the implementation, since I figured future releases won't need owners approval. Regardless, my implementation has approval now, so I'll just create 0.0.1 and re-send that.